eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/io/MergedVariantMongoReader.java

```diff
@@ -135,11 +138,19 @@ private String getInactiveField(String field) {
 
                 addToVariants(variants, contig, submittedVariantStart, rs, reference, alternate, sourceEntry);
             }
+
+            if (submittedVariantOperation.getString(EVENT_TYPE_FIELD).equals(EventType.UPDATED.toString())
+                    && submittedVariantOperation.getString(REASON).startsWith(DECLUSTER_OPERATION_REASON_FIRST_WORD)) {
+                hasSubmittedVariantsDeclustered = true;
+            }
         }
 
-        if (variants.isEmpty()) {
-            throw new IllegalStateException("There was a merge operation for rs" + rs + " but there is no update " +
-                                                    "operation for the SS IDs.");
+        if (!hasSubmittedVariantsDeclustered && variants.isEmpty()) {
+            throw new IllegalStateException ("There was a merge operation for rs" + rs + " but there is no update " +
+                                                     "operation for the SS IDs. This can be either an error or a " +
+                                                     "special scenario where all the corresponding SS IDs were " +
```

Comment:
Isn't hasSubmittedVariantsDeclustered supposed to indicate precisely this condition?

Reply:
You are right! Message have been updated